src/Project/functions/copyItem.ts

```diff
@@ -5,8 +5,18 @@ import { isCompilableFile } from "Project/util/isCompilableFile";
 import { DTS_EXT } from "Shared/constants";
 
 export function copyItem(data: ProjectData, pathTranslator: PathTranslator, item: string) {
+	// Check if source file/directory exists before attempting to copy
+	if (!fs.pathExistsSync(item)) {
+		return;
+	}
```

Comment:
These comments aren't super helpful - comments should explain _why_, not (just) what. With this explanation, one might think `copyItem` expects to receive files that never existed in the first place, but that's not the intention. (The comment should explain that the check guards race conditions.)

I'm not sure this approach would fully solve the race condition. We don't have a file handle on `item`, so there is still a period between `pathExistsSync` and `copySync` that the file could be deleted. It'd be better to handle the not found error in the result of `copySync`.